plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -48,13 +62,16 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Delete
             client.delete(deleteRequest, new ActionListener<DeleteResponse>() {
                 @Override
                 public void onResponse(DeleteResponse deleteResponse) {
-                    log.info("Completed Delete Model Request, model id:{} deleted", modelId);
-                    actionListener.onResponse(deleteResponse);
+                    deleteModelChunks(modelId, deleteResponse, actionListener);
+                    log.info("Completed Delete Model Request for model id:{} ", modelId);
```

Comment:
This log may mislead user, as delete model chunks may fail later. Remove this line?

Reply:
Removed in the next commit. I thought it was fine to keep the line since it only means the request is processed. After a second thought, we are better off getting rid of it.